cypress/cypress/e2e/01_submariner_deployment_validation.cy.js

```diff
@@ -22,7 +22,16 @@ describe('submariner - Deployment validation', {
     
     it('verify that a cluster set with deployment of submariner is exists.', { tags: ['verify'] }, function () {
         let clusterSetName = Cypress.env('CLUSTERSET')
-        submarinerClusterSetMethods.submarinerClusterSetShouldExist(clusterSetName)
+        submarinerClusterSetMethods.submarinerClusterSetShouldExist(clusterSetName).then(($clusterset) => {
+            if ($clusterset.text().includes(clusterSetName)) {
+                cy.log(clusterSetName + " was found")
```

Comment:
Could you add a "clusterset" to the string so we could understand what was or wasn't found?
Like:
```
cy.log(clusterSetName + " clusterset was found")
```

Here and in lines below.